WORD_to_OSK_ID_mapping.md

```diff
@@ -0,0 +1,93 @@
+# WORD to `OSK_ID` Mapping
```

Comment:
I'd rather move this file to `docs/WORD mapping.md`

Reply:
good idea :D